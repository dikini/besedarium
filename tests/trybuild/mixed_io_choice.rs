use playground::*;

type MixedIOChoice = tchoice!(Http;
    TInteract<Http, TClient, Message, TEnd<Http>>,
    TInteract<Mqtt, TBroker, Publish, TEnd<Mqtt>>
);
