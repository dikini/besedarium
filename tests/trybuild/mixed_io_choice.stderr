error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/trybuild/mixed_io_choice.rs:6:3
  |
6 | );
  |   ^ consider adding a `main` function to `$DIR/tests/trybuild/mixed_io_choice.rs`
